inc/admin/network/class-sharingandprivacyoptions.php

```diff
@@ -207,6 +208,73 @@ public function updateBooksLastUpdatedDate() {
 		}
 	}
 
+	/**
+	 * Performs network book directory exclusion logic for non catalog books
+	 * @param bool $exclude   True for exclude and false for removing exclude
+	 */
+	public static function networkExcludeOption( bool $exclude ) {
```

Comment:
Can be simplified by self::excludeNonCatalogBooksFromDirectory( 'excludeNonCatalogBooksFromDirectoryAction', $exclude );

Reply:
will be revised.